### ISSUE TITLE
cant acces home/user folder

### ISSUE THREAD

**Issue by juan11perez** [coderaiser/cloudcmd#178]
* **Version** (cloudcmd -docker image:latest): 
* **OS** (Ubuntu server 18.04): 
* **Browser chrome latest: 
* **Changed Config**: no

first of all thank you for this package. Im running on docker and works well; however, I cant access my home/users folder. 
It is loaded because i can access it when going into the container, but on the we-browser i get nothing. 

Does it require additional/special configuration?

**Comment by coderaiser** [comment]
Is your issue resolved already? What was the case?

**Comment by juan11perez** [comment]
Thank you for your reply. I did get it working and there was no issue. Just lack of understanding on my part. I did however have a major system crash and can’t really point it, but it happened when I pxoxied your container.

I’m using the Linuxserver/ letsEncrypt  container and have 12 apps exposed.

When I added your container to the sever location block and restarted nginx it crashed and didn’t start anymore.  I removed the block and then I got errors from docker compose, but I hadn’t touched it.

Finally I got it back up, but the whole set up was messed up. Kept getting “not accessible” messages and ended up reinstalling everything again



I rub Ubuntu 18.04 Server LTS
